WooCommerce/Classes/ViewRelated/Products/Media/ProductImagesViewController.swift

```diff
@@ -127,9 +132,25 @@ private extension ProductImagesViewController {
     }
 
     func configureHelperViews() {
-        helperContainerView.isHidden = allowsMultipleImages || product.productType != .variable
         helperLabel.applySecondaryFootnoteStyle()
-        helperLabel.text = Localization.variableProductHelperText
+        updateHelperViews()
+    }
+
+    var shouldShowHelperViews: Bool {
+        return getAppropiateHelpText() != nil
+    }
+
+    /// Returns the appropiate localizable help text or `nil` when
+    /// there is no help text to show.
+    ///
+    func getAppropiateHelpText() -> String? {
+        if productImageStatuses.containsMoreThanOne {
+            return Localization.dragAndDropHelperText
+        } else if !(allowsMultipleImages || product.productType != .variable) {
```

Comment:
Mmm...it seems very similar but it's not exactly the same. For example, in this case, the result is different:

```swift
var allowsMultipleImages = true
var productType = ProductType.variable

!(allowsMultipleImages || productType != .variable) // false

!allowsMultipleImages || productType == .variable // true
```

BTW, it's true that this sentence is hard to read 😕 I could create a computed var to encapsulate this logical operation, something like `shouldShowOnlyOnePhotoHelperText`. WDYT?

Reply:
> the result is different

My bad! The reverse of that statement should be:

```Swift
!allowsMultipleImages && productType == .variable
```

Is that correct or am I mistaken again? 😅

Reply:
> something like `shouldShowOnlyOnePhotoHelperText`

I think that naming would not be sufficient because the content is specific to product variations only. I'd say we keep the condition check for simplicity. How does that sound?

Reply:
> Is that correct or am I mistaken again? 😅

Now you are correct 🙂

> I'd say we keep the condition check for simplicity

 I agree with you 👍🏼 Let me change it

Reply:
Done ✅